topo/node/node.go

```diff
@@ -220,6 +225,66 @@ func (n *Impl) Create(ctx context.Context) error {
 	return nil
 }
 
+func CombineErrors(errors []error) string {
+	var combinedErrorString string
+	for _, err := range errors {
+		if combinedErrorString != "" {
+			combinedErrorString += "\n"
+		}
+		combinedErrorString += err.Error()
+	}
+	return combinedErrorString
+}
+
+func (n *Impl) ValidateConstraints() []error {
+	hostConstraints := n.GetProto().GetHostConstraints()
+	if hostConstraints == nil {
+		return nil
+	}
+	var errorList []error
```

Comment:
use the errlist library instead, should make this easier: https://github.com/openconfig/kne/blob/fd11a07f35a7afb429ecc511400c1f53e5ad3d6b/deploy/deploy.go#L174

Reply:
Done.